common/src/main/kotlin/org/waste/of/time/storage/cache/HotCache.kt

```diff
@@ -67,12 +71,41 @@ object HotCache {
      * @param chunkZ The Z coordinate of the chunk.
      * @return True if the chunk is saved, false otherwise.
      */
+    @Deprecated("This method will default to the current dimension. Please use the new method by passing in a dimension.")
     @Suppress("unused")
-    fun isChunkSaved(chunkX: Int, chunkZ: Int) = savedChunks.contains(ChunkPos.toLong(chunkX, chunkZ))
+    fun isChunkSaved(chunkX: Int, chunkZ: Int) = isChunkSaved(chunkX, chunkZ, mc.world?.registryKey!!)
```

Comment:
world can be null, make this fall back to overworld: `mc.world?.registryKey ?: World.OVERWORLD`